### PR TITLE
WMCO 1.0.6 release notes

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: https://deploy-preview-40977--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes#wmco-1-0-6

### CODE REVIEW

**Comment by codyhoag** [comment]
@rrasouli can you verify? Thanks!

File: windows_containers/windows-containers-release-notes.adoc [hunk 1]
```diff
@@ -17,6 +17,20 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
...
+Issued: 2022-02-14
+
+The WMCO 1.0.6 is now available with a security fix. The components of the WMCO were released in link:https://access.redhat.com/errata/RHSA-2022:0240[RHSA-2022:0240].
+
+The support statements and known issues documented for the xref:../windows_containers/windows-containers-release-notes.adoc#wmco-1-0-2[WMCO 1.0.2] release also apply for this release of the WMCO.
+
+[id="wmco-1-0-6-bug-fixes"]
+=== Bug fixes
+
+* A flaw was found in the way Unicode standards are implemented in the context of development environments, which have specialized requirements for rendering text. An attacker could exploit this to deceive a human reviewer by creating a malicious patch containing well placed BiDi characters. The special handling and rendering of those characters could have been used to hide unexpected and potentially dangerous behavior from the reviewer. This security issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2005819[*BZ#2005819*])
```
Comment (ctauchen) [inline windows_containers/windows-containers-release-notes.adoc:32]: A flaw was found in the way Unicode standards are implemented in the context of development environments, which have specialized requirements for rendering text.

"that have specialized"?? Unless it's all development environments.

Maybe: s/BiDi/bidirectional

s/well placed/well-placed per https://www.merriam-webster.com/dictionary/well-placed

**Review by ctauchen — COMMENTED** [review]
A few things, otherwise LGTM.

Not familiar with this component -- is this really only for 4.6?

**Pull request merged**